modules/templates-writing-parameters.adoc

```diff
@@ -63,13 +63,14 @@ characters long consisting of all upper and lowercase alphabet letters
 and numbers.
 
 The syntax available is not a full regular expression syntax. However, you can
-use `\w`, `\d`, and `\a` modifiers:
+use `\w`, `\d`, `\a`, and `\A` modifiers:
```

Comment:
Should be `\\w`, `\\d`, `\\a`, and `\\A`

Reply:
Fixed; see next comment for link to updates. Thanks a lot for the guidance.

Reply:
Heya @xiuwang, 

Pinging again as I wanted to get this ticket correct. Can you verify Sam's comments below?